docs/search/get-started-with-search/search-page/index.md

```diff
@@ -64,6 +64,12 @@ In this section, we'll introduce the following concepts:
   <p>Learn how to filter results based on a histogram time range.</p>
   </div>
 </div>
+<div className="box smallbox card">
+  <div className="container">
+  <a href="/docs/search/get-started-with-search/search-page/event-analyticsl"><img src={useBaseUrl('img/icons/search.png')} alt="icon" width="40"/><h4>Event Analytics</h4></a>
+  <p>Learn more about events in histogram and aggregate charts for your search results.</p>
+  </div>
+</div>
```

Comment:
Move down after "Field Browser".

Reply:
Doc is made as Beta, so we will not update the index page.